src/renderer/components/Editor/TakeGroupComponent.tsx

```diff
@@ -55,8 +73,25 @@ const TakeGroupComponent = ({
   transcription,
   ...passThroughProps
 }: TakeGroupComponentProps) => {
+  const [takeDissolved, setTakeDissolved] = useState(false);
```

Comment:
Do we actually need this? Isn't `takeDissolved` implied by the take group being deleted, and then this wouldn't be rendered any more?

Reply:
Yeah originally the components were still being rendered and I didn't know why. I've had a further look and found that the takeDetection.ts generated TakeGroup TranscriptionChunks under the assumption that take group ids always start from 0 and increase by 1 (which obviously doesn't support deleted take groups). I have just fixed it in this last commit and have also removed the useState.

Reply:
nice catch